src/metabase/lib/aggregation.cljc

```diff
@@ -2,10 +2,187 @@
   (:refer-clojure :exclude [count distinct max min])
   (:require
    [metabase.lib.common :as lib.common]
+   [metabase.lib.metadata.calculation :as lib.metadata.calculation]
    [metabase.lib.options :as lib.options]
+   [metabase.lib.schema :as lib.schema]
+   [metabase.lib.schema.aggregation :as lib.schema.aggregation]
+   [metabase.lib.schema.common :as lib.schema.common]
+   [metabase.lib.util :as lib.util]
+   [metabase.shared.util.i18n :as i18n]
    [metabase.util.malli :as mu])
   #?(:cljs (:require-macros [metabase.lib.aggregation])))
 
+(mu/defn resolve-aggregation :- ::lib.schema.aggregation/aggregation
```

Comment:
Moved some of the existing code that was in `metabase.lib.metadata.calculate.resolve` to places that actually need/use it